apps/screloc/main.cpp

```diff
@@ -503,6 +503,12 @@ class Reloc : public Client::Application {
 
 				ci->setAgencyID(agencyID());
 				ci->setAuthor(author());
+
+				// keep track of the triggering origin of this relocation
+				DataModel::CommentPtr comment = new DataModel::Comment();
+				comment->setId("screlocSourceOrigin");
```

Comment:
The comment can also hold a version and an author, so basically a `CreationInfo`. Maybe populating one of these with `name()` could be helpful, too.

I know that comment ids are arbitrary, but maybe it could be beneficial later to use some encoding. We decided to separate the scope/app with colon from the specifier, e.g. `screloc:sourceOrigin`.